package.json

```diff
@@ -45,10 +45,12 @@
     "prismjs": "^1.6.0",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
+    "react-is": "^16.8.6",
     "react-live": "^2.1.2",
-    "styled-components": "^4.0.0-beta.11",
```

Comment:
Love how we only upgrade styled-components when testing betas 🤣

Reply:
Lol right? 😆